rs/replicated_state/tests/replicated_state.rs

```diff
@@ -911,19 +911,19 @@ proptest! {
                 output_iter.exclude_queue();
                 excluded += 1;
             } else {
-                assert_eq!(Some(msg.clone()), output_iter.next());
+                prop_assert_eq!(Some(msg.clone()), output_iter.next());
                 consumed += 1;
             }
         }
 
         drop(output_iter);
-        assert_eq!(total_requests, excluded + consumed);
-        assert_eq!(replicated_state.output_message_count(), excluded);
+        prop_assert_eq!(total_requests, excluded + consumed);
+        prop_assert_eq!(replicated_state.output_message_count(), excluded);
     }
 
     #[test]
     fn iter_yields_correct_elements(
-       (mut replicated_state, mut raw_requests, _total_requests) in arb_replicated_state_with_queues(SUBNET_ID, 20, 20, None),
```

Comment:
Why are we reducing from 20 to 10 everywhere?

Reply:
I don't think 20 gives us anything that 10 doesn't.

You only need a handful of canisters with a handful of queues with some randomness thrown in to test whether iteration order is respected or not. If there's a bug in how we pop a queue from the front and push it to the back, you'll either catch it with 3 canisters / 3 queues or not catch it at all (because it requires some sequence of events that cannot be triggered by your proptest -- e.g. because you never `peek()` twice in succession).

Reply:
Thanks! Makes sense.